src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -471,8 +471,9 @@ public partial class DataRowView : System.ComponentModel.ICustomTypeDescriptor,
         object System.ComponentModel.ICustomTypeDescriptor.GetPropertyOwner(System.ComponentModel.PropertyDescriptor pd) { throw null; }
 #nullable enable
     }
-    [System.ComponentModel.DesignerAttribute("Microsoft.VSDesigner.Data.VS.DataSetDesigner, Microsoft.VSDesigner, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
     [System.ComponentModel.DefaultPropertyAttribute("DataSetName")]
+    [System.ComponentModel.DesignerAttribute("Microsoft.VSDesigner.Data.VS.DataSetDesigner, Microsoft.VSDesigner, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+    [System.ComponentModel.ToolboxItemAttribute(false)]
```

Comment:
This looks different than the value in the framework: https://referencesource.microsoft.com/#System.Data/fx/src/data/System/Data/DataSet.cs,40

Reply:
Good catch. Will update.

Reply:
After seeing this I went back and checked all of the others and couldn't find any other diffs.

Reply:
Yeah, this was the only one I got wrong for some reason.

Reply:
Done